cypress/features/regression/multiActionButton.feature

```diff
@@ -42,4 +42,12 @@ Feature: Multi Action Button default component
     Given I open "Multi Action Button Test" component page "default"
       And clear all actions in Actions Tab
     When I click on "button"
-    Then click action was called in Actions Tab
\ No newline at end of file
+    Then click action was called in Actions Tab
+
+  @positive
```

Comment:
Maybe it'll worth adding test cases to open hidden container by Tabbing/ focusing button ?

Reply:
I'd only provided one case here to test that when the Multi Action Button is opened in a hidden container, an Accordion in this case, the dropdown options are visible. It had been cut off previously. Opening an Accordion itself with a click or Enter key is tested in the Accordion feature file so I didn't think it was necessary here too.